addon/mixins/spread.js

```diff
@@ -46,20 +46,6 @@ export default Mixin.create({
     // State
   },
 
-  // == Computed Properties ===================================================
-  _sourceChanged: observer(`${SPREAD_PROPERTY}`, 'spreadOptions.source.object.options',
-    function () {
-      const {propertyPath, spreadSource} = this._getSpreadSource()
-      if (spreadSource === undefined) {
-        this._resetSpreadProperties()
-      } else if (spreadSource.setUnknownProperty === undefined) {
-        this._resetSpreadProperties()
-        this._addSetUnsupportedProperty(propertyPath)
-      }
-      if (spreadSource) {
-        this._defineSpreadProperties(propertyPath, spreadSource)
-      }
-    }),
```

Comment:
need to use `addObserver` instead of `observer(...)` because there's a scenario where property is dynamic.